src/content/en/updates/posts/2016/05/houdini.md

```diff
@@ -0,0 +1,366 @@
+draft is one of the more mature ones and a [polyfill][Typed CSSOM polyfill] is
+actually already being worked on (Disclaimer: Using the polyfill will obviously
+add *even more* computational overhead. The point is to show how convenient the
+API is).
+
+Instead of strings you will be working on an element’s `StylePropertyMap`, where
+each CSS attribute has it’s own key and corresponding value type. Attributes
+like `width` have `LengthValue` as their value type. A `LengthValue` is a
+dictionary of all CSS units like `em`, `rem`, `px`, `percent`, etc. Setting
+`height: calc(5px + 5%)` would yield a `LengthValue{px: 5, percent: 5}`. Some
+properties like `box-sizing` just accept certain keywords and therefore have a
+`KeywordValue`. The validity of those attributes could now be checked at
+runtime.
+
+{% highlight JS %}
+var w1 = $('#div1').styleMap.get('width');
+var w2 = $('#div2').styleMap.get('width');
+$('#div3').styleMap.set('background-size',
+  [new SimpleLength(200, 'px'), w1.add(w2)])
+$('#div4')).styleMap.get('margin-left')
+  // => {em: 5, percent: 50}
+{% endhighlight %}
+
+### Font Metrics
+
+Font metrics is exactly what it sounds like. What is the bounding box (or the
+bounding boxes when we are wrapping) when I render string X with font Y at size
+Z? What if I go all crazy unicode on you like using [ruby annotations]? This has
+been requested a lot in the past and Houdini should finally make these wishes
+come true.
+
+### Properties and Values ([spec][Properties and Values spec])
+
+Do you know CSS Custom Properties (or their unofficial alias “CSS Variables”)?
+This is them but with types! So far, variables could only have string values and
+used a simple search-and-replace approach. This draft would allow you to not
+only specify a type for your variables, but also define a default value and
+influence the inheritance behavior using a JavaScript API. Technically, this
+would also allow custom properties to get animated with standard CSS transitions
+and animations, which is being considered.
+
+{% highlight JS %}
+["--scale-x", "--scale-y"].forEach(function(name) {
+document.registerProperty({
+    name: name,
+    syntax: "<number>",
+    inherits: false,
+    initialValue: "1"
+  });
+});
+{% endhighlight %}
+
+### But wait, there’s more!
+
+There’s even more specs in Houdini’s list of drafts, but the future of those is
+rather uncertain and they are not much more than a placeholder for an idea at
+this point. Examples include custom overflow behaviors, CSS syntax extension
+API, extension of native scroll behavior and similar ambitious things that allow
+you to screw up your web app even worse than ever before. Rejoice!
+
+## Gimme!
+
+As of now, nothing of the Houdini specs is in Chrome. However, some of them may
+be available behind a “OMG this is totally not for production” flag soon(tm) in
+Canary.
+
+For what it’s worth, I have open-sourced the [code for the demo][Houdini
```

Comment:
I thought about this, but since no browser has support for the Houdini APIs *yet*, I thought it might be rather misleader. How about we add the links once we have support in at least some of the unstable builds?